tv_apps/api-demos-tv/index.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <meta name="viewport" content="width=device-width, user-scalable=no, initial-scale=1">
+  <meta charset="utf-8">
+  <title>UI tests</title>
+  <!-- We don't need localization in this app -->
+  <link rel="stylesheet" type="text/css" href="style/uitest.css">
+  <link rel="stylesheet" type="text/css" href="style/tabs.css" >
+  <script type="text/javascript" src="js/uitest.js"></script>
+  <script defer src="shared/js/lazy_loader.js"></script>
+  <script defer src="shared/js/accessibility_helper.js"></script>
+</head>
+<body>
```

Comment:
It should be. But this version is copied from UItest. This version doesn't support keyboard operation. I will file a bug to convert the this version to support keyboard.